### PR TITLE
Enabled getting only the most recent logged event info

### PR DESCRIPTION
**Pull request by jontje**
As per title.

Solves issue https://github.com/ros-industrial/abb_librws/issues/53.

### CODE REVIEW

File: src/rws_client.cpp [hunk 1]
```diff
@@ -644,6 +644,11 @@ std::string RWSClient::getLogText(const bool verbose)
   return ss.str();
 }
 
+std::string RWSClient::getLogTextLatestEvent(const bool verbose)
+{
+  return (log_.size() == 0 ? "Log is empty." : log_[0].toString(verbose, 0));
```
Comment (gavanderhoorn) [inline src/rws_client.cpp]: I would suggest to return an empty string if the log is empty.

And document that `getLogTextLatestEvent(..)` returns an empty string if the log is empty.

That should be sufficient and an empty string makes sense if there are no log messages.

Comment (jontje) [reply src/rws_client.cpp]: Good idea, I have pushed the changes.

**Comment by gavanderhoorn** [comment]
Thanks @jontje.

**Comment by jontje** [comment]
Thanks for the quick review and merge @gavanderhoorn 😄

**Pull request merged**